xapian-letor/bin/questletor.cc

```diff
@@ -70,28 +63,30 @@ static void show_usage() {
     cout << "Usage: " PROG_NAME " [OPTIONS] 'QUERY'\n"
 "NB: QUERY should be quoted to protect it from the shell.\n\n"
 "Options:\n"
-"  -d, --db=DIRECTORY  database to search (multiple databases may be specified)\n"
-"  -m, --msize=MSIZE   maximum number of matches to return\n"
-"  -s, --stemmer=LANG  set the stemming language, the default is 'english'\n"
+"  --db=DIRECTORY  database to search (multiple databases may be specified)\n"
+"  --model=PATH  path to model file to use in ranking\n"
+"  --msize=MSIZE   maximum number of matches to return\n"
+"  --stemmer=LANG  set the stemming language, the default is 'english'\n"
 "                      (pass 'none' to disable stemming)\n"
-"  -p, --prefix=PFX:TERMPFX  Add a prefix\n"
-"  -b, --boolean-prefix=PFX:TERMPFX  Add a boolean prefix\n"
-"  -h, --help          display this help and exit\n"
-"  -v, --version       output version information and exit\n";
+"  --prefix=PFX:TERMPFX  Add a prefix\n"
+"  --boolean-prefix=PFX:TERMPFX  Add a boolean prefix\n"
+"  --help          display this help and exit\n"
+"  --version       output version information and exit\n";
 }
 
 int
 main(int argc, char **argv)
 try {
-    const char * opts = "d:m:s:p:b:hv";
+    const char * opts = "a:b:c:d:e:f:g:h";
```

Comment:
I think it's better to have short options. I'd normally say that whichever is more likely to be used should get `-m`, so it's more guessable. However here I wonder if we can use `-f` for "model file", since `-f` is so often used to specify a file. (And `--model-file`, perhaps.)